src/main/java/com/softserve/kickscootertrip/model/entity/TripEntity.java

```diff
@@ -0,0 +1,35 @@
+package com.softserve.kickscootertrip.model.entity;
+
+import lombok.Data;
+import org.springframework.data.geo.Point;
+
+import javax.persistence.*;
+import java.util.Date;
+import java.util.UUID;
+
+@Data
+@Entity
+@Table(name = "trip")
+public class TripEntity {
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @Column(name = "tripId")
+    private UUID tripId;
+    @Column(name = "userId")
+    private UUID userId;
+    @Column(name = "scooterId")
+    private UUID scooterId;
+    @Column(name = "start")
+    private Date start;
+    @Column(name = "finish")
+    private Date finish;
+    @Column(name = "tripTime")
+    private long tripTime;
+    @Column(name = "point")
+    private Point point;
```

Comment:
don't know, but shoudn't it be onetomany dependency, i.e. trip should persist full route(list of points)?

Reply:
it seems so